app/services/foreman_deprecation.rb

```diff
@@ -0,0 +1,24 @@
+class ForemanDeprecation
```

Comment:
Namespace this beneath Foreman:: rather than simply prefixing it - i.e. app/services/foreman/deprecation.rb.